frontend/test/metabase/scenarios/moderation/question-moderation.cy.spec.js

```diff
@@ -50,6 +50,17 @@ describeWithToken("scenarios > saved question moderation", () => {
         .find(".Icon-verified")
         .should("not.exist");
     });
+
+    it("should be able to see evidence of verification/unverification in the question's timeline", () => {
+      cy.findByTestId("saved-question-header-button").click();
+      cy.findByText("History").click();
+
+      cy.findByTestId("moderation-verify-action").click();
+      cy.findByText("You verified this").should("exist");
+
+      cy.findByTestId("moderation-remove-review-action").click();
+      cy.findByText("You unverified this").should("exist");
```

Comment:
I guess we prefer to avoid using `should("exist")` explicitly, as `findBy*` is an assertion itself

Reply:
removed `should("exist")`